src/modules/Elsa.MongoDB/Features/MongoDbFeature.cs

```diff
@@ -0,0 +1,117 @@
+using System.Security.Authentication;
+using Elsa.Features.Abstractions;
+using Elsa.Features.Services;
+using Elsa.Http.Models;
+using Elsa.Identity.Contracts;
+using Elsa.Identity.Entities;
+using Elsa.Labels.Contracts;
+using Elsa.Labels.Entities;
+using Elsa.MongoDB.HostedServices;
+using Elsa.MongoDB.Modules.Identity;
+using Elsa.MongoDB.Modules.Labels;
+using Elsa.MongoDB.Modules.Management;
+using Elsa.MongoDB.Modules.Runtime;
+using Elsa.MongoDB.Options;
+using Elsa.MongoDB.Serializers;
+using Elsa.Workflows.Core.Services;
+using Elsa.Workflows.Core.State;
+using Elsa.Workflows.Management.Contracts;
+using Elsa.Workflows.Management.Entities;
+using Elsa.Workflows.Runtime.Contracts;
+using Elsa.Workflows.Runtime.Entities;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using MongoDB.Bson.Serialization;
+using MongoDB.Bson.Serialization.Serializers;
+using MongoDB.Driver;
+using MongoDB.Driver.Core.Extensions.DiagnosticSources;
+using StoredBookmark = Elsa.Workflows.Runtime.Entities.StoredBookmark;
+using WorkflowExecutionLogRecord = Elsa.Workflows.Runtime.Entities.WorkflowExecutionLogRecord;
+
+namespace Elsa.MongoDB.Features;
+
+/// <summary>
+/// Configures MongoDb.
+/// </summary>
+public class MongoDbFeature : FeatureBase
+{
+    /// <inheritdoc />
+    public MongoDbFeature(IModule module) : base(module)
+    {
+    }
+
+    /// <summary>
+    /// A delegate that configures MongoDb.
```

Comment:
In comments, when referring to brands etc, we should use the real name: "MongoDB".